src/components/widgets/status/StatusTab.vue

```diff
@@ -435,4 +435,52 @@ export default class StatusTab extends Mixins(StateMixin, FilesMixin) {
     direction: rtl;
     text-align: left;
   }
+  .position-relative {
+  position: relative;
+}
+
+.progress-wrapper {
+  position: relative;
+  width: 90px; /* Set the width and height to match progress circle size */
+  height: 90px; /* Set the width and height to match progress circle size */
+  overflow: hidden;
+  border-radius: 50%;
+}
+
+.progress-circle {
+  position: absolute;
+  z-index: 1;
+}
+
+.thumbnail-background {
+  position: absolute;
+  width: 80%; /* 10% less than the size of the progress circle */
+  height: 80%; /* 10% less than the size of the progress circle */
+  border-radius: 50%;
+  background-color: rgba(0, 0, 0, 0.5); /* Adjust transparency here */
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 0;
+}
+
+.print-thumb {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 0;
+  max-height: 70px; /* Adjust as needed */
+  border-radius: 50%;
+  opacity: 0.8; /* Adjust opacity as needed */
+}
+
+.percentage-wrapper {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-43%,200%);
+
+}
```

Comment:
The indentation here is off - and please add a blank line before this section and remove the one after

Reply:
Not sure why the lint didn't catch that. will do.